### ISSUE TITLE
List all nodes

### ISSUE THREAD

**Issue by abdou31** [yinguobing/butterfly#1]
Hello Yin,
Thanks for this great tool, 
I list all the nodes and this is the output of command line : 
> python3 demo.py --model frozen_inference_graph3.pb --list_ops true

> butterfly/OneShotIterator
butterfly/IteratorGetNext
butterfly/input_to_float
butterfly/conv2d/kernel
butterfly/conv2d/kernel/read
butterfly/conv2d/bias
butterfly/conv2d/bias/read
butterfly/conv2d/Conv2D
butterfly/conv2d/BiasAdd
butterfly/conv2d/Relu
butterfly/max_pooling2d/MaxPool
butterfly/conv2d_1/kernel
butterfly/conv2d_1/kernel/read
butterfly/conv2d_1/bias
butterfly/conv2d_1/bias/read
butterfly/conv2d_1/Conv2D
butterfly/conv2d_1/BiasAdd
butterfly/conv2d_1/Relu
butterfly/conv2d_2/kernel
butterfly/conv2d_2/kernel/read
butterfly/conv2d_2/bias
butterfly/conv2d_2/bias/read
butterfly/conv2d_2/Conv2D
butterfly/conv2d_2/BiasAdd
butterfly/conv2d_2/Relu
butterfly/max_pooling2d_1/MaxPool
butterfly/conv2d_3/kernel
butterfly/conv2d_3/kernel/read
butterfly/conv2d_3/bias
butterfly/conv2d_3/bias/read
butterfly/conv2d_3/Conv2D
butterfly/conv2d_3/BiasAdd
butterfly/conv2d_3/Relu
butterfly/conv2d_4/kernel
butterfly/conv2d_4/kernel/read
butterfly/conv2d_4/bias
butterfly/conv2d_4/bias/read
butterfly/conv2d_4/Conv2D
butterfly/conv2d_4/BiasAdd
butterfly/conv2d_4/Relu
butterfly/max_pooling2d_2/MaxPool
butterfly/conv2d_5/kernel
butterfly/conv2d_5/kernel/read
butterfly/conv2d_5/bias
butterfly/conv2d_5/bias/read
butterfly/conv2d_5/Conv2D
butterfly/conv2d_5/BiasAdd
butterfly/conv2d_5/Relu
butterfly/conv2d_6/kernel
butterfly/conv2d_6/kernel/read
butterfly/conv2d_6/bias
butterfly/conv2d_6/bias/read
butterfly/conv2d_6/Conv2D
butterfly/conv2d_6/BiasAdd
butterfly/conv2d_6/Relu
butterfly/max_pooling2d_3/MaxPool
butterfly/conv2d_7/kernel
butterfly/conv2d_7/kernel/read
butterfly/conv2d_7/bias
butterfly/conv2d_7/bias/read
butterfly/conv2d_7/Conv2D
butterfly/conv2d_7/BiasAdd
butterfly/conv2d_7/Relu
butterfly/flatten/Shape
butterfly/flatten/strided_slice/stack
butterfly/flatten/strided_slice/stack_1
butterfly/flatten/strided_slice/stack_2
butterfly/flatten/strided_slice
butterfly/flatten/Reshape/shape/1
butterfly/flatten/Reshape/shape
butterfly/flatten/Reshape
butterfly/dense/kernel
butterfly/dense/kernel/read
butterfly/dense/bias
butterfly/dense/bias/read
butterfly/dense/MatMul
butterfly/dense/BiasAdd
butterfly/dense/Relu
butterfly/logits/kernel
butterfly/logits/kernel/read
butterfly/logits/bias
butterfly/logits/bias/read
butterfly/logits/MatMul
butterfly/logits/BiasAdd

How can I know the output nodes and input nodes names?

**Comment by yinguobing** [comment]
Most of the time, the input and output node names should be provided by the model's author. If none provided, you will have to try it out. In the output above, I have not found any reasonable input node name, and the output node name I guess is `butterfly/logits/BiasAdd`.

**Comment by abdou31** [comment]
I tried the three node for input and the last three nodes for output and I got many arrays of values.

I guess that the three node is the input node( input_to_float) it was writed in `landmark.py` as an input for image.
In addition, when I write the three first nodes I get error.that the two first nodes are not in.the graph.
